src/main/User.java

```diff
@@ -12,10 +12,10 @@
     Date lastLoginDate;
 
     public ActionResult login(String username, String password) {
-        String pass = password.hashCode();
+        int pass = password.hashCode();
```

Comment:
in ghesmat haye user ro ham ke avaz krdi bargrdoon chon okeyesh krdm conflict mishe